Fix changeset publish

Our recent Version Package PRs that have been merged into main have failed the publish step:
https://github.com/interledger/open-payments/commits/main/

Most recently [here](https://github.com/interledger/open-payments/actions/runs/8740656125/job/23984830784):

```
> @ ci:publish /home/runner/work/open-payments/open-payments
> pnpm changeset publish

🦋  info npm info @interledger/http-signature-utils
🦋  info npm info @interledger/open-payments
🦋  info npm info @interledger/openapi
🦋  warn @interledger/http-signature-utils is not being published because version 2.0.2 is already published on npm
🦋  warn @interledger/openapi is not being published because version 2.0.1 is already published on npm
🦋  info @interledger/open-payments is being published because our local version (6.9.0) has not been published on npm
🦋  info Publishing "@interledger/open-payments" at "6.9.0"
🦋  error an error occurred while publishing @interledger/open-payments: E404 Not Found - PUT https://registry.npmjs.org/@interledger%2fopen-payments - Not found 
🦋  error 
🦋  error  '@interledger/open-payments@6.9.0' is not in this registry.
🦋  error 
🦋  error Note that you can also install from a
🦋  error tarball, folder, http url, or git url.
🦋  error npm notice Publishing to https://registry.npmjs.org/ with tag latest and public access
🦋  error npm ERR! code E404
🦋  error npm ERR! 404 Not Found - PUT https://registry.npmjs.org/@interledger%2fopen-payments - Not found
🦋  error npm ERR! 404 
🦋  error npm ERR! 404  '@interledger/open-payments@6.9.0' is not in this registry.
🦋  error npm ERR! 404 
🦋  error npm ERR! 404 Note that you can also install from a
🦋  error npm ERR! 404 tarball, folder, http url, or git url.
🦋  error 
🦋  error npm ERR! A complete log of this run can be found in: /home/runner/.npm/_logs/[20](https://github.com/interledger/open-payments/actions/runs/8740656125/job/23984830784#step:4:21)24-04-18T15_29_06_194Z-debug-0.log
🦋  error 
🦋  error packages failed to publish:
🦋  @interledger/open-payments@6.9.0
 ELIFECYCLE  Command failed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "format:docs": "prettier --write ./docs/**/*.md",
     "changeset:add": "pnpm changeset add",
     "ci:version": "pnpm changeset version && pnpm i --lockfile-only",
-    "ci:publish": "pnpm changeset publish",
+    "ci:publish": "pnpm publish -r",
     "checks": "pnpm check:prettier && pnpm check:lint",
     "check:lint": "eslint --max-warnings=0 .",
     "check:prettier": "prettier --check .",

